apps/web/src/views/universalFarms/components/PositionItem.tsx

```diff
@@ -211,6 +236,7 @@ export const PositionV3Item = memo(({ data, pool }: IPositionV3ItemProps) => {
 })
 
 interface IPositionItemDetailProps {
+  chainId: number
   currencyBase?: Currency
```

Comment:
can we just use currency0 & currency1

as amm common appoint, token1 always quote

Reply:
done